solutions/usaco-712.mdx

```diff
@@ -160,7 +160,72 @@ For every cow (say cow $N$), we can check which cows have one entry/exit point b
 **Time Complexity:** $\mathcal{O}(T^3)$, where $T$ represents the total number of characters in Farmer John's string.
 
 <LanguageSection>
-	
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+const int TOTAL = 52;
```

Comment:
quick comment explaining what this means

Reply:
Line 167 to 169? 
In question the length of string is fixed (26 * 2 = 56) so I decided to add a const variable representing total number of characters.